content/en/profiler/connect_traces_and_profiles.md

```diff
@@ -75,13 +75,22 @@ os.Setenv("DD_PROFILING_EXECUTION_TRACE_PERIOD", "15m")
 
 Setting these variables will record up to 1 minute (or 5 MiB) of execution tracing data [every 15 minutes][2].
 
-While recording execution traces, your application may observe an increase in CPU usage similar to a garbage collection. Although this should not have a significant impact for most applications, Go 1.21 includes [patches][3] to eliminate this overhead.
+You can find this data:
+
+- In the [Profile List][3] by adding `go_execution_traced:yes` to your search query. Click on a profile to view the [Profile Timeline][4]. To go even deeper, download the profile and use `go tool trace` or [gotraceui][5] to view the contained `go.trace` files.
+- In the [Trace Explorer][6] by adding `@go_execution_traced:yes` (note the `@`) to your search query. Click on a span and then select the `Code Hotspots` tab to view the [Span Timeline](#span-execution-timeline-view).
+
+While recording execution traces, your application may observe an increase in CPU usage similar to a garbage collection. Although this should not have a significant impact for most applications, Go 1.21 includes [patches][7] to eliminate this overhead.
```

Comment:
I realize this line hasn't actually changed, but I don't see any reference to what an "execution trace" is in this documentation. Should we provide a link? And maybe a quick reference to what it is in-line, like "While recording execution traces, the data powering Go timelines, ..."?

Reply:
Sure! I'm happy to do that